### PR TITLE
Setup/server

### PR DESCRIPTION
**Pull request by enriq018**
**Basic server setup**

added express router and '/customerReviews' endpoint
added server test using supertest, mocha, and chai

### CODE REVIEW

File: server/routes/customerReviews.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const express = require("express");
```
Comment (beth) [inline server/routes/customerReviews.js]: Check linter settings (and consider a precommit hook) - these should be single quotes.

File: spec/server/app.spec.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const expect = require('chai').expect;
+const supertest = require('supertest');
+
+const api = supertest('http://localhost:3005');
+
+describe('User', () => {
+  it('should return 200', (done) => {
+    api.get('/customerReviews')
+    .set('Accept', 'Application/json')
```
Comment (beth) [inline spec/server/app.spec.js]: What does this line do? (just curious)

File: package.json [hunk 3]
```diff
@@ -18,5 +19,19 @@
   "bugs": {
     "url": "https://github.com/NotAsBadAsItSteams/customerReviews/issues"
   },
-  "homepage": "https://github.com/NotAsBadAsItSteams/customerReviews#readme"
+  "homepage": "https://github.com/NotAsBadAsItSteams/customerReviews#readme",
+  "dependencies": {
+    "body-parser": "^1.18.2",
```
Comment (beth) [inline package.json]: Do you use this? If not, I suggest removing it until you do.

File: package.json [hunk 4]
```diff
@@ -4,7 +4,8 @@
   "description": "customer reviews for steam toast",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (beth) [inline package.json]: Non-blocking, but I'm guessing this should be `mocha spec` something similar - probably a good idea to add it to save you the hassle of typing it all the time.

**Review by beth — CHANGES_REQUESTED** [review]
@enriq018 Just one needed change (style problems) but there are a few other comments to just consider as well.

File: server/routes/customerReviews.js (hunk 1 above)
Comment (enriq018) [reply server/routes/customerReviews.js]: My linter somehow broke midway through and i had no idea. Reinstalled it and will look into precommit hooks. Thank you!

File: spec/server/app.spec.js (hunk 2 above)
Comment (enriq018) [reply spec/server/app.spec.js]: Updated this test with a better description and removed the .set.
It makes a get request to /customerReviews and checks to see if status code 200 is received.

File: package.json (hunk 3 above)
Comment (enriq018) [reply package.json]: Removed!

**Review by beth — APPROVED** [review]

**Pull request merged**